app/views/coc/finished.scala.html

```diff
@@ -38,5 +38,9 @@ <h1 class="heading-large"><img src="@routes.Assets.at("images/tick.png")" alt=""
 <p>@Html(Messages("pages.coc.finish.junk"))</p>
 <h2 class="heading-medium">@Html(Messages("pages.coc.finish.whn"))</h2>
 <p>@Html(Messages("pages.coc.finish.para1"))</p>
-<p>@Html(Messages("pages.coc.finish.para2"))</p>
+<p>
+ @messages("pages.coc.finish.para2.part1")
```

Comment:
Needs HTML tag as the welsh message includes an ascii code. Alternatively, replace the ascii code with its character equivalent. Same for all instances of this message key

Reply:
replaced ascii code